.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -0,0 +1,50 @@
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**Security Impact (if applicable)**
+- [ ] This issue might have security implications
```

Markdownlint (0.35.0)

Lists should be surrounded by blank lines

(MD032, blanks-around-lists)